visualization/app/codeCharta/ui/codeMap/codeMap.component.scss

```diff
@@ -1,2 +1,21 @@
 code-map-component {
+
+  view-cube-component {
```

Comment:
Why did you have to move these lines from the `view-cube-component`?

Reply:
I think about it like this: If I want to have multiple view-cubes / reuse a component, I want to apply style that defines it's position and look in the container, that it's placed in. The view-cube has no internal structure, that's why the css in the viewCube.component.scss is empty.

Let me give you a better example:
You have a container with two buttons in it and named it container.component.ts
If you want to position the buttons inside this container, you'd add the css code to container.component.scss
If you want to position the container inside your application, you'd move the css-code one layer up. This way, you can reuse the container everywhere and only have to apply css for the outer container. The buttons inside the container will be styled correctly all the time, because the css is one layer down.

Reply:
Alright, yea that makes sense 👍